scripts/test-wp-cs.sh

```diff
@@ -216,28 +228,62 @@ do_client_test() { # usage: do_client_test [extraArgs]
 }
 
 FAIL=0
+
+WOLFPROV_DIR=$PWD
 WOLFPROV_NAME="libwolfprov"
-WOLFPROV_PATH=$PWD/.libs
+WOLFPROV_PATH=$WOLFPROV_DIR/.libs
 
 CURVES=prime256v1
 #CURVES=X25519
 OPENSSL_ALL_CIPHERS="-cipher ALL -ciphersuites $TLS13_ALL_CIPHERS"
 OPENSSL_PORT=$(generate_port)
 
 init_openssl
+init_wolfssl
 if [ -z $LD_LIBRARY_PATH ]; then
     export LD_LIBRARY_PATH="$OPENSSL_INSTALL_DIR/lib64:$WOLFSSL_INSTALL_DIR/lib"
 else
     export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$OPENSSL_INSTALL_DIR/lib64:$WOLFSSL_INSTALL_DIR/lib"
 fi
 printf "LD_LIBRARY_PATH: $LD_LIBRARY_PATH\n"
 
-printf "\tClient testing\n" | tee $LOG_FILE
+# Set up wolfProvider
+cd ${WOLFPROV_DIR}
+if [ ! -e "${WOLFPROV_DIR}/configure" ]; then
+    ./autogen.sh 2>&1 >> $LOG_FILE
+    ./configure --with-openssl=${OPENSSL_INSTALL_DIR} --with-wolfssl=${WOLFSSL_INSTALL_DIR} 2>&1 >> $LOG_FILE
```

Comment:
```
In scripts/test-wp-cs.sh line 253:
    ./autogen.sh 2>&1 >> $LOG_FILE
                 ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).
                         ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```
```
In scripts/test-wp-cs.sh line 254:
    ./configure --with-openssl=${OPENSSL_INSTALL_DIR} --with-wolfssl=${WOLFSSL_INSTALL_DIR} 2>&1 >> $LOG_FILE
                               ^--------------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
                                                                     ^--------------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
                                                                                            ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).
                                                                                                    ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```